contracts/btc-staking/src/queries.rs

```diff
@@ -296,25 +296,29 @@ mod tests {
         let fps = crate::queries::finality_providers(deps.as_ref(), None, None)
             .unwrap()
             .fps;
+
         let fp1 = FinalityProvider::from(&new_fp1);
         let fp2 = FinalityProvider::from(&new_fp2);
-        assert_eq!(fps.len(), 2);
-        assert_eq!(fps[0], fp1);
-        assert_eq!(fps[1], fp2);
+        let expected_fps = vec![fp1.clone(), fp2.clone()];
+        assert_eq!(fps.len(), expected_fps.len());
+        for fp in fps {
+            assert!(expected_fps.contains(&fp));
+        }
```

Comment:
An alternative here and below is to first sort both arrays before comparing them.

Reply:
Tried, but somehow the test still failed, probably the results are returned out of order. That's why I ended up simply checking inclusion